### ISSUE TITLE
Build instructions fail at cmake on Ubuntu 18.04

### ISSUE THREAD

**Issue by bensulli** [bedrock-viz/bedrock-viz#58]
On Ubuntu 18.04 I install the pre-requisites and run the build.md commands (I just make them a script, so they're verbatim), but it results in:

```
-- Build type: Release
You have called ADD_LIBRARY for library pugixml without any source files. This typically indicates a problem with your CMakeLists.txt file
You have called ADD_LIBRARY for library pugixml-shared without any source files. This typically indicates a problem with your CMakeLists.txt file
You have called ADD_LIBRARY for library pugixml-static without any source files. This typically indicates a problem with your CMakeLists.txt file
-- Found PNG: /usr/lib/x86_64-linux-gnu/libpng.so (found version "1.6.34")
-- Boost version: 1.65.1
-- Found the following Boost libraries:
--   program_options
CMake Error at CMakeLists.txt:69 (target_link_libraries):
  Object library target "libbedrock-viz" may not link to anything.


-- Configuring incomplete, errors occurred!
See also "/home/deepsummer/bedrock-viz/build/CMakeFiles/CMakeOutput.log".
See also "/home/deepsummer/bedrock-viz/build/CMakeFiles/CMakeError.log".
make: *** No targets specified and no makefile found.  Stop.
```

I was able to get this working with the same steps on Ubuntu 19.10, however.

**Comment by paulgrahek** [comment]
Thank you for finding this error. It should be resolved with #60

**Comment by bpr-sulli** [comment]
Awesome! I can confirm this has resolved the issue and commented on the PR (sadly I'm not qualified to review the code itself).
